pkg/schedule/checker/checker_controller.go

```diff
@@ -125,6 +120,14 @@ func (c *Controller) CheckRegion(region *core.RegionInfo) []*operator.Operator {
 			c.regionWaitingList.Put(region.GetID(), nil)
 		}
 	}
+	// only consider merge checker when region label is set to "schedule=deny".
+	if cl, ok := c.cluster.(interface{ GetRegionLabeler() *labeler.RegionLabeler }); ok {
```

Comment:
maybe putting it into code line 137  be better?

Reply:
This is not just for merge, but only merge is left. I change the comments.